www/views/timers.html

```diff
@@ -177,7 +177,7 @@ <h2>{{ ::'Name'|translate }}: {{::vm.itemName}}</h2>
                 </select>
             </td>
         </tr>
-        <tr id="TimersLedColor" ng-show="::vm.isColorSettingsAvailable" class="ng-hide">
+        <tr id="TimersLedColor" ng-show="::vm.isColorSettingsAvailable" class="ng-show">
```

Comment:
@stas-demydiuk I changed this since the rgbww picker does not render correctly when hidden, is that OK?

Reply:
@emontnemery , no this is not the way. You have to either make it visible by letting the routine "vm.isColorSettingsAvailable" return true, or remove the whole ng-show attribute when it should 'allways' be visible

Reply:
@gizmocuz  The problem is that vm.isColorSettingsAvailable initially returns true **after** the color picker has already been created (initLEDLightSettings function in app/timers/components.js:136). Thus, the colorpicker is created with zero size which causes display bugs.
By setting the element to default visible instead of default invisible there are no display bugs.

Reply:
@gizmocuz There is a proper fix in #2815, can this be merged first? Otherwise I can revert this change.

Reply:
Sure!